### DISCUSSION TITLE
Change language

### DISCUSSION THREAD

**Discussion by bameyrick** [ONSdigital/design-system#1391] [Feedback]
Use this place to discuss the [change language pattern](https://ons-design-system.netlify.com/patterns/change-language) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by RhodGriffith** [comment]
The current language value in the drop down menu (4 language option) is Arial rather than Lato. 
<img width="212" alt="screen shot 2019-02-06 at 07 46 49" src="https://user-images.githubusercontent.com/28897806/52327465-27e34980-29e4-11e9-91b7-b364ab83259e.png">

**Comment by dudelmeister** [comment]
Highlight (focus)

Should the focus state be the same as the 'Select' Component (because they look similiar)? 

or as it is now (with padding like a button)?

**Comment by bameyrick** [comment]
> Highlight (focus)
> 
> Should the focus state be the same as the 'Select' Component (because they look similiar)?
> 
> or as it is now (with padding like a button)?

@will-madebymoon what do you think?

**Comment by bameyrick** [comment]
If you haven't focused the dropdown version and then long click, the menu will open from focusing but  it will then register the click event when the mouse is released and close the menu.

**Comment by bameyrick** [comment]
As discussed with @will-madebymoon, we should remove the JavaScript dropdown version of the language selector as this failed massively in DAC testing. Mocking a dropdown isn't accessible despite using the correct aria attributes.

**Comment by jrbarnes9** [comment]
Wrong link to Design Forum in 'Change language' pattern

**Comment by jrbarnes9** [comment]
Language toggle links should only display in footer if they're not displayed in the header

![Screenshot_20201031-080939](https://user-images.githubusercontent.com/43346934/97774600-201cee80-1b51-11eb-94c0-88a0323518a8.jpg)

**Comment by elthorne** [comment]
Hello, where do I raise any problems with the translation itself please? I can confirm the attached is incorrect and should be:

Cysylltwch a ni 
Cwcis a phreifatrwydd

Additionally the "a" in "Cysylltwch a ni" needs a circumflex which I'm not sure how to input.

<img width="250" alt="welsh translation" src="https://user-images.githubusercontent.com/46898565/154922746-3e960185-422d-4a66-a5fb-f1bdbaef2b0c.PNG">

**Reply by jrbarnes9** [reply]
Raised #1995

**Comment by jrbarnes9** [comment]
Language selector macro options aren't visible on the website. 

This is because the component isn't documented as a standalone component. It is imported by the Header component, so these options should be documented within the header macro options file.

![Screenshot 2023-01-23 at 17 10 31](https://user-images.githubusercontent.com/43346934/214104198-7196d78b-9bca-46f5-8786-307098a2e6d8.png)

**Reply by jrbarnes9** [reply]
Raised #2585
